### ISSUE TITLE
Make face aligment optional

### ISSUE THREAD

**Issue by roperi** [mantasu/face-crop-plus#5]
It would be nice to have a way to turn off the face aligment to avoid using padding. Especially good for training models where the position of the face/body is not important.

**Comment by roperi** [comment]
For example, fine-tuning of pre-trained diffusion models where you actually want to preserve face or body position.

**Comment by mantasu** [comment]
Yes, thanks for this! Will consider in v1.2

**Comment by BarbokOL** [comment]
Hey, no Idea how active this still is, but I would love to still see this feature and was wondering if there will be any more work or if I should look for an alternative?
